Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -90,6 +92,27 @@ public void ProcRunner_FailsOnTimeout()
             System.Threading.Thread.Sleep(1100);
         }
 
+        [TestMethod]
+        public void ProcRunner_PassesEnvVariables()
+        {
+            // Arrange
+            string exeName = WriteBatchFileForTest(
+@"echo %PROCESS_VAR%
```

Comment:
The product code is expected to handle setting multiple environment variables so the test should supply multiple values.
There are multiple possible errors in the product code that this test wouldn't catch e.g. only setting the first/last value; e.g. correctly looping through the variables, but using the same key each time.